### PR TITLE
added maintenance note

### PR DESCRIPTION
**Pull request by SamuelMR98**

### CODE REVIEW

**Comment by SamuelMR98** [comment]
#25 Added Maintenance Note

**Comment by SamuelMR98** [comment]
#16 Added Maintenance Note

File: migrations/0001_initial_creation.up.sql [hunk 1]
```diff
@@ -46,5 +46,6 @@ CREATE TABLE issue_events (
 CREATE TABLE room_maintenance_couch (
 	couch_room_id text PRIMARY KEY,
 	start_time timestamptz IS NOT NULL,
-	end_time timestamptz IS NOT NULL
+	end_time timestamptz IS NOT NULL,
```
Comment (bwinterton) [inline migrations/0001_initial_creation.up.sql:49]: I've already got these changes pushed up in master, can we remove these?

File: website/src/app/api.service.ts [hunk 2]
```diff
@@ -196,7 +197,7 @@ export class ApiService {
         if (info?.end) {
           info.end = new Date(info.end);
         }
-
+        console.log(info)
```
Comment (bwinterton) [inline website/src/app/api.service.ts:200]: Let's remove this console.log

**Review by bwinterton — CHANGES_REQUESTED** [review]
This all looks awesome! There's just a couple of things to change and then we'll get it merged in.

File: migrations/0001_initial_creation.up.sql [hunk 3]
```diff
@@ -47,5 +47,4 @@ CREATE TABLE room_maintenance_couch (
 	couch_room_id text PRIMARY KEY,
 	start_time timestamptz IS NOT NULL,
 	end_time timestamptz IS NOT NULL,
```
Comment (bwinterton) [inline migrations/0001_initial_creation.up.sql:49]: Remove trailing comma here.

**Review by bwinterton — CHANGES_REQUESTED** [review]

**Review by bwinterton — APPROVED** [review]
LGTM

**Pull request merged**